src/Honeybee_Honeybee.py

```diff
@@ -5022,7 +5025,116 @@ def assignScheduleBasedOnProgram(self, component = None):
         
         # find all the patameters and assign them to 
         self.isSchedulesAssigned = True
+        
+    def getConstructionsByStandardClimateZone(self,standard,climateZone,component=None):
+        
+        openStudioStandardLib = sc.sticky["honeybee_OpenStudioStandardsFile"]
+        
+        try:
+            schedulesAndLoads = openStudioStandardLib['space_types']['90.1-2007']['ClimateZone 1-8'][self.bldgProgram][self.zoneProgram]
+            
+        except:
+            msg = "Either your input for bldgProgram > [" + self.bldgProgram + "] or " + \
+                  "the input for zoneProgram > [" + self.zoneProgram + "] is not valid.\n" + \
+                  "Use ListSpacePrograms component to find the available programs."
+            print msg
+            if component != None:
+                component.AddRuntimeMessage(gh.GH_RuntimeMessageLevel.Warning, msg)
+                
+        # Get construction set,
+        
+        constructionSet = openStudioStandardLib['construction_sets'][standard][climateZone][self.bldgProgram]
+        
+        return constructionSet
```

Comment:
What happens to the schedules and loads? You're just returning the constructionSet here? Also you should add a try/exception for this case too.